src/App.jsx

```diff
@@ -146,67 +178,58 @@ function App() {
-              </a>
-              <i
-                className="bx bxs-chevron-down arrow"
-                onClick={() => {
-                  setIsFileMenuOpen(!isFileMenuOpen);
-                }}
-              ></i>
-            </div>
+          <li className='show-menu'>
             <ul className="sub-menu">
-              <li>
-                <a className="link-name" href="#">
-                  Files
-                </a>
+              <li className='files'>
+                <div className='full-Box'>
+                  <div>
+                    <a className='link-name' href="#">
+                      Files
+                    </a>
+                  </div>
+                  <div>
+                    <i class='bx bxs-file-plus newBTN' style={{ color: '#dbdde1' }} onClick={() => { createFile() }} ></i>
+                  </div>
+                </div>
               </li>
               {data
                 ? Object.keys(data).map((key) => (
-                    <li
-                      key={key}
-                      onClick={() => {
-                        setActiveFileId(key);
-                        if (!tabList.includes(key)) {
-                          setTabList([...tabList, key]);
-                        }
+                  < li className="file" >
+                    <div className='full-Box' onClick={() => {
+                      setActiveFileId(key);
+                      if (!tabList.includes(key)) {
+                        setTabList([...tabList, key]);
+                      }
+                    }}
+                      onMouseEnter={() => {
+                        setUtils(true);
                       }}
-                      onDoubleClick={() => {
-                        renameFile(key);
-                      }}
-                    >
-                      <a>{data[key].name}</a>
-                    </li>
-                  ))
+                      onMouseLeave={() => {
+                        setUtils(false);
+                      }}>
+                      <div>
+                        <a className='name'>{data[key].name}</a>
+                      </div>
+                      <div className={utils ? 'utils' : 'noUtils'}>
+                        <i className='bx bxs-edit utilities' style={{ color: '#abdafb' }} onClick={() => { renameFile(key) }}></i>
+                        <i className='bx bx-trash' style={{ color: '#eb336c' }} onClick={() => { deleteFile(key) }}></i>
+                      </div>
+                    </div>
+
+                  </li>
+                ))
                 : null}
             </ul>
           </li>
           <li>
-            <div className="profile-details">
+            <div className="settings">
               <a href="#">
-                <i className="bx bx-cog"></i>
-                <span className="link-name">Setting</span>
+                <span className='settings' onClick={() => setSettings(!settings)}>Settings</span>
```

Comment:
The parent `div` and child `span` have the same class, which might lead to some confusion.